src/Application/Actions/Hooks/StripePayoutUpdate.php

```diff
@@ -37,6 +37,18 @@ public function __construct(
     ) {
         parent::__construct($container, $logger);
         $this->chatterFactory = $chatterFactory;
+
+        /**
+         * @var array{
+         *    notifier: array{
+         *      slack: array{
+         *        stripe_channel: string
+         *      }
+         *    }
+         *  } $settings
+         */
+        $settings = $container->get('settings');
+        $this->slackStripeChannel = $settings['notifier']['slack']['stripe_channel'];
```

Comment:
would it be worth injecting the slackStripeChannel into the constructor instead of pulling it from the container here?

Reply:
@bdsl Ideally yes, but building custom DI support for specific settings is a bigger undertaking.

What I'd like to do is refactor to use a proper settings class as introduced in newer Slim skeleton versions, but I don't have time to do this today. For example, ID is using this at https://github.com/thebiggive/identity/blob/develop/app/settings.php#L13